postgresql-native/build.gradle

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+plugins {
+    id 'java'
+    id 'com.github.spotbugs'
+    id 'checkstyle'
+    id 'jacoco'
+}
+
+description = 'Ballerina - PostgreSQL Java Native'
+
+configurations {
+    jacocoRuntime
+}
+
+dependencies {
+    jacocoRuntime "org.jacoco:org.jacoco.agent:${jacoco.toolVersion}:runtime"
+    compile files(configurations.externalJars)
```

Comment:
```suggestion
    compile "org.postgresql:postgresql:${postgreSQLDriverVersion}"
```

Reply:
I will update

Reply:
Updated.
Commit id is c58731650a7387ff7c9687737fe5ec9280a2f2de